src/main/java/nl/dtls/fairdatapoint/entity/exception/CommunicationException.java

```diff
@@ -0,0 +1,15 @@
+package nl.dtls.fairdatapoint.entity.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.BAD_REQUEST)
+@Getter
+@AllArgsConstructor
+public class CommunicationException extends RuntimeException {
+
+    private final String message;
```

Comment:
This is quite a generic name, would there be a case for renaming this to reflect the specific case (I assume shape importing in the scope of this PR at least)?

Reply:
Is that better now? 😄 I wanted to rename to `cause` but it is not possible with `RuntimeException`...